test/TEST-40-NBD/test.sh

```diff
@@ -225,7 +225,7 @@ make_encrypted_root() {
         inst_binary "${basedir}/dracut-util" "/usr/bin/dracut-util"
         ln -s dracut-util "${initdir}/usr/bin/dracut-getarg"
         ln -s dracut-util "${initdir}/usr/bin/dracut-getargs"
-
+        inst_simple ./99-default.link /etc/systemd/network/99-default.link
```

Comment:
Is this still needed with the modules.d/35network-legacy/module-setup.sh change?

Same for the rest of the changes.

Reply:
Yup this was still needed for that test after the network-legacy change. ( did not revert the changes in the other test to check if they worked or not ).